presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleAssert.java

```diff
@@ -54,15 +55,17 @@
     private final TransactionManager transactionManager;
     private final AccessControl accessControl;
     private final StatsCalculator statsCalculator;
+    private final CostCalculator costCalculator;
 
-    public RuleAssert(Metadata metadata, Session session, Rule rule, TransactionManager transactionManager, AccessControl accessControl, StatsCalculator statsCalculator)
+    public RuleAssert(Metadata metadata, Session session, Rule rule, TransactionManager transactionManager, AccessControl accessControl, StatsCalculator statsCalculator, CostCalculator costCalculator)
     {
         this.metadata = metadata;
         this.session = session;
         this.rule = rule;
         this.transactionManager = transactionManager;
         this.accessControl = accessControl;
         this.statsCalculator = statsCalculator;
+        this.costCalculator = costCalculator;
```

Comment:
Can this be `null`?

(I _see_ other fields are not validated, this is sad :( )